src/cli/mod.rs

```diff
@@ -0,0 +1,11 @@
+mod config;
+use crate::api_client;
+
+pub use config::{BeaconMethod, CliConfig, Namespace::Beacon};
```

Comment:
I think we only need to make the `CliConfig` `pub`:

```suggestion
use config::{BeaconMethod, Namespace::Beacon};
pub use config::CliConfig;
```

and I'd hold off until we have more stuff here but it may ultimately be easiest to just say `use config::*;`